CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Changelog
 
-# Upcoming
+# 1.4.0
+
+## Features
+- **M3Query**: write endpoint support for M3-Map-Tags-JSON header in influxdb ([#3816](https://github.com/m3db/m3/pull/3816))
```

Comment:
nit: Capitalize first word for consistency? “Write endpoint” or even “Add write endpoint support”..